pkg/resources/sts/sts.go

```diff
@@ -85,3 +87,38 @@ func GetSTSCredentialsFromEnvVar() (string, string, error) {
 
 	return roleARN, tokenPath, nil
 }
+
+// GetS3BucketCredentials retrieves Credential addon parameters to be used by 3scale
+// Using S3 Bucket Credentiol addon parameters is temporary solution,
+// workaround for the lack of STS support in 3Scale,
+// as noted in https://issues.redhat.com/browse/MGDAPI-1905,
+// until our feature request https://issues.redhat.com/browse/THREESCALE-7132 gets implemented
+func GetS3BucketCredentials(ctx context.Context, client k8sclient.Client, namespace string) (string, string, error) {
```

Comment:
This function doesn't seem used, are we keeping it for some reason ? :thinking:

Reply:
Done. Thank you very much for review and comments